### DISCUSSION TITLE
How to get List of parameters in rules and run contains operation on them

### DISCUSSION THREAD

**Discussion by bhanu-affinidi** [hyperjumptech/grule-rule-engine#297] [Q&A]
I want to have list of parameters as rules definition and when I rule is executed I want to run contains operator on top of this.

**Comment by newm4n** [comment]
I dont really get your question, could you explain to us "list of parameters as rules definition" ? What are you trying todo ?

**Reply by bhanu-affinidi** [reply]
I want to have an array of values in rule definition and when I call that rule with one value , rule should check if provided value is part of array defined in rule definition

**Reply by newm4n** [reply]
Hmm... you could always create your own function in a struct and add the struct to data context. The function will check any argument you provide toward an internal array. Or, you can hard code each values checking as in `when a == 1 || a == 2 || a == 3 then ... " kind of way.
